mw-webapp/src/component/select/Select.tsx

```diff
@@ -163,7 +174,7 @@ export const Select = <T extends string>(props: SelectProps<T>) => {
   return (
     <div
       className={clsx(styles.select, props.className)}
-      data-cy={props.cy?.dataCyOverlay}
+      data-cy={`${props.cy?.dataCyOverlay || ""} ${props.dataCy || ""}`.trim()}
```

Comment:
Looks a bit strange. Please check  is it right?

Reply:
fixed